src/main/java/Duke/Task/TaskList.java

```diff
@@ -0,0 +1,79 @@
+package Duke.Task;
+
+import Duke.Storage.Storage;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Maybe it would be good to include header comments for this class and its methods since it is an integral part of this program?